tests/test_deps.py

```
@@ -36,31 +36,33 @@
 except Exception as _cause:  # pragma: no branch
 	raise ImportError("[CWE-440] venv Failed to import.") from _cause
 
 
+@context.markWithMetaTag("extra", "extra_coverage")
 class BuildRequirementsTxtTestSuite(context.BasicUsageTestSuite):
 	"""Test cases for 'tests/requirements.txt'."""
 
-	__module__ = """tests.test_deps"""
+	__module__ = "tests.test_deps"
+
+	def test_requirements_path_exists(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_requirements_path_exists`

Add return type annotation: `None`

(ANN201)